modules/con_notifications-configuration.adoc

```diff
@@ -92,6 +92,31 @@ spec:
 <1> Default name of the `NotificationsConfiguration` CR in a cluster.
 <2> An example custom service configuration for the `NotificationsConfiguration` CR.
 
+The following two examples are used for referencing the `oauth-access-token` in the `argocd-notifications-secret`. These steps are optional. 
+
+.Example for storing `oauth-access-token` in the `argocd-notifications-secret`:
+[source,yaml]
+----
+  apiVersion: v1
+  kind: Secret
+  metadata:
+      name: <secret-name>
+  stringData:
+      slack-token: <oauth-access-token>
+----
+
+.Example for creating a configmap to reference the `oauth-access-token` in the `argo-notifications-secret`:
+[source,yaml]
+----
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: argocd-notifications-cm
+  data:
+    service.slack: |
+      token: $slack-token
```

Comment:
Consider adding callouts to explain the use of these two lines.

Reply:
I agree with Max here as these seem like custom values. Is the $ sign needed? I have not seen this in a YAML before. Is "slack-token" an environment variable (EV)? EVs are usually capitalized. I'd recommend asking the engineer about this.